src/main/java/com/purbon/kafka/topology/actions/access/builders/rbac/BuildBindingsForSchemaAuthorization.java

```diff
@@ -13,11 +13,18 @@ public class BuildBindingsForSchemaAuthorization extends BaseAccessControlAction
 
   private final BindingsBuilderProvider builderProvider;
   private final Schemas schemaAuthorization;
+  private final Boolean shouldOptimizeAcls;
+  private final String namePrefix;
 
   public BuildBindingsForSchemaAuthorization(
-      BindingsBuilderProvider builderProvider, Schemas schemaAuthorization) {
+      BindingsBuilderProvider builderProvider,
+      Schemas schemaAuthorization,
+      Boolean shouldOptimizeAcls,
+      String namePrefix) {
```

Comment:
another small question: I wonder if we can find a better name for this variable? should we better speak about schemaName prefix? having only name might be confusing.

what do you think?